v2/clients/http/deviceprofile.go

```diff
@@ -44,3 +46,31 @@ func (client *DeviceProfileClient) Update(ctx context.Context, reqs []requests.D
 	}
 	return responses, nil
 }
+
+func (client *DeviceProfileClient) DeleteById(ctx context.Context, id string) (common.BaseResponse, errors.EdgeX) {
```

Comment:
I thought we drop byID APIs except for cases where there isn't appropriate byName to use.

Reply:
Removed.
You're right, thanks for catching that, I just add the implemented API.